app/scripts/modules/kubernetes/v2/serverGroup/details/resize/resize.html

```diff
@@ -0,0 +1,54 @@
+<div modal-page class="confirmation-modal">
+  <task-monitor monitor="ctrl.taskMonitor"></task-monitor>
+  <form role="form" name="resizeForm">
+    <modal-close dismiss="$dismiss()"></modal-close>
+    <div class="modal-header">
+      <h3>Resize {{ctrl.serverGroup.kind}} {{ctrl.serverGroup.displayName}}</h3>
+    </div>
+    <div class="modal-body confirmation-modal">
+      <div class="form-horizontal">
+        <div class="form-group form-inline">
+          <div class="col-md-3 sm-label-right">
+            Current
+          </div>
+          <div class="col-md-3">
+            <div class="input-group">
+              <input type="number"
+                     class="form-control input-sm highlight-pristine"
+                     ng-model="ctrl.current.desired"
+                     readonly/>
+              <span class="input-group-addon">pod<span ng-if="ctrl.current.desired != 1">s</span></span>
```

Comment:
Nothing is going to go wrong here, since the input type is `number`, but should be a `!==` here and below.

Reply:
Ah good catch